coredumper/coredumper.py

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env python3
+
```

Comment:
Can we have some comments here on what this script does and basic usage (more usage can go in --help and in sof-docs) Btw, probably also best to rename to sof-core ?.